app/views/alaveteli_pro/subscriptions/_card.html.erb

```diff
@@ -1,30 +1,35 @@
-<%= card.brand %>
-<%= card.last4 %>
-<%= card.exp_month %>/<%= card.exp_year %>
-<%= card_expiry_message(card.exp_month, card.exp_year) %>
+<div class="settings__item">
+  <div class="settings__item__primary">
+    <span class="card-desc"><strong><%= card.brand %></strong> <%= _('ending in') %> <%= card.last4 %><%= _(', expiry') %> <%= card.exp_month %>/<%= card.exp_year %></span>
+  </div>
+  <div class="settings__item__secondary">
+    <%= form_tag(update_subscription_payment_path) do %>
+
+    <%= hidden_field_tag 'old_card_id', card.id %>
 
-<%= form_tag(update_subscription_payment_path) do %>
+    <script src="https://checkout.stripe.com/checkout.js" class="stripe-button"
+            data-key="<%= AlaveteliConfiguration.stripe_publishable_key %>"
+            data-name="<%= AlaveteliConfiguration.pro_site_name %>"
+            data-panel-label="Update Card Details"
+            data-label="Update Card Details"
+            data-email="<%= current_user.email %>"
+            data-locale="auto"
+            data-allow-remember-me=false
+            data-image="https://s3.amazonaws.com/stripe-uploads/acct_19EbqNIbP0iBLddtmerchant-icon-1479145884111-mysociety-wheel-logo.png"
+            data-zip-code=false>
+    </script>
 
-  <%= hidden_field_tag 'old_card_id', card.id %>
+    <noscript>
+      <div id="error">
+        <p>
+          <%= _('Updating your payment card requires your browser to have ' \
+                'JavaScript enabled.') %>
+        </p>
+      </div>
+    </noscript>
+  </div>
+<%= card_expiry_message(card.exp_month, card.exp_year) %>
+</div>
 
-  <script src="https://checkout.stripe.com/checkout.js" class="stripe-button"
-          data-key="<%= AlaveteliConfiguration.stripe_publishable_key %>"
-          data-name="<%= AlaveteliConfiguration.pro_site_name %>"
-          data-panel-label="Update Card Details"
-          data-label="Update Card Details"
-          data-email="<%= current_user.email %>"
-          data-locale="auto"
-          data-allow-remember-me=false
-          data-image="https://s3.amazonaws.com/stripe-uploads/acct_19EbqNIbP0iBLddtmerchant-icon-1479145884111-mysociety-wheel-logo.png"
-          data-zip-code=false>
-  </script>
 
-  <noscript>
-    <div id="error">
-      <p>
-        <%= _('Updating your payment card requires your browser to have ' \
-              'JavaScript enabled.') %>
-      </p>
-    </div>
-  </noscript>
 <% end %>
```

Comment:
Oh I've also just noticed that the closing tag of this form looks like its going to wrap the outer div, so that'll need fixing

Reply:
I think the form should end just after the `</script>`

Reply:
oh, cheers will fix